src/bsp/include/bsp/IInterlocManager.h

```diff
@@ -9,6 +12,12 @@ class IInterlocManager {
      * @brief Dummy function to demonstrate working DW1000s
      */
     virtual void startInterloc() = 0;
+
+    /**
+     * @brief Registers a callback to be called when new interloc data is available
+     * @param callback Callback to call
+     */
+    virtual void registerPositionUpdateCallback(std::function<void(InterlocUpdate)> callback) = 0;
```

Comment:
Maybe declare a typedef for the function callback? Might simplify integration to have a known function definition for the callback instead of having to check the argument passed

Reply:
Changed for a `using` (as suggested by clang instead of typedef)